dist/berlin-transport-card.js

```diff
@@ -31,15 +33,20 @@ class BerlinTransportCard extends HTMLElement {
             }
 
             const timetable = entity.attributes.departures.slice(0, maxEntries).map((departure) => {
-                const delay = departure.delay === 'null' ? `` : departure.delay / 60;
-                const delay_div = delay > 0 ? `<div class="delay delay-pos">+${delay}</div>`: `<div class="delay delay-neg">${delay === 0 ? '+0' : delay}</div>`;
-                return departure.cancelled && !showCancelled ? `` :
+            const delay = departure.delay === 'null' ? `` : departure.delay / 60;
+            const delayDiv = delay > 0 ? `<div class="delay delay-pos">+${delay}</div>`: `<div class="delay delay-neg">${delay === 0 ? '+0' : delay}</div>`;
+            const currentDate = new Date().getTime();
+            const timestamp = new Date(departure.timestamp).getTime();
+            const relativeTime = Math.round((timestamp - currentDate) / (1000 * 60)) - departure.walking_time;
```

Comment:
Why you decided to subtract walking_time here? I think it will be confusing, for example, if the train will be in 5 minutes but the widget shows only "2". Or did I misunderstand this construction?

Reply:
It's the time to leave to reach the bus in time

Reply:
I can add another option not to include it

Reply:
I've added another option to include the walking time or not.